TYPO3.Flow/Classes/TYPO3/Flow/Security/GlobalObjectStateInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Created by IntelliJ IDEA.
+ * User: sebastian
+ * Date: 17.03.17
+ * Time: 09:58
+ */
+
+namespace TYPO3\Flow\Security;
+
+interface GlobalObjectStateInterface
```

Comment:
figured as much, however this should be documented somehow in https://github.com/neos/flow-development-collection/blob/master/Neos.Flow/Documentation/TheDefinitiveGuide/PartIII/Security.rst